docs/source/cc_launcher.md

```diff
@@ -185,7 +185,7 @@ chaincode:
     path: /builders/binary
 ```
 
-In this example, the implementation of "my-golang-builder" is contained within the `/builders/golang` directory and its build scripts are located in `/builders/golang/bin`. When the peer invokes any of the build scripts associated with "my-golang-builder", it will propagate only the values of the environment variables in the whitelist.
+In this example, the implementation of "my-golang-builder" is contained within the `/builders/golang` directory and its build scripts are located in `/builders/golang/bin`. When the peer invokes any of the build scripts associated with "my-golang-builder", it will propagate only the values of the environment variables in the `environmentWhitelist`.
```

Comment:
allowList, maybe?

Reply:
Changing this requires a breaking change in the code.

Maybe we could just say: `it will propagate only the environment variables specified in the externalBuilder config`

Reply:
https://jira.hyperledger.org/browse/FAB-18028

Reply:
You could pass the call through, and retire the old interface over time?

Reply:
Thanks @sykesm